ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -340,4 +349,22 @@ public void shouldCloseInternallyCreatedTopicClientOnClose() {
     // Then:
     verify(topicClient);
   }
+
+
+  @Test
+  public void shouldParseMultipleStatements() throws IOException {
+    final String statementsString = new String(Files.readAllBytes(
+        Paths.get("src/test/resources/SampleMultilineStatements.sql")), "UTF-8");
+
+    final List<Pair<String, Statement>> parsedStatements =
+        ksqlEngine.parseStatements(statementsString, new MetaStoreImpl(new TestFunctionRegistry()));
```

Comment:
can `new MetaStoreImpl(new TestFunctionRegistry())` be replaced by `metaStore`?

Reply:
When we use the query file the metastore is empty so it's better to test it with the expected situation, empty metastore.

Reply:
Don't know what exactly you mean by `emptyMetaStore`, but made it a variable named `emptyMetaStore` and pass it to the method call.